cirq/google/known_devices.py

```diff
@@ -57,7 +57,7 @@ def _parse_device(s: str) -> Tuple[List[GridQubit], Dict[str, Set[GridQubit]]]:
 
 def create_device_proto_from_diagram(
         ascii_grid: str,
-        gate_set: serializable_gate_set.SerializableGateSet = None,
+        gate_sets: Iterable[serializable_gate_set.SerializableGateSet] = None,
```

Comment:
nit: Should be `Optional[Iterable[...]]` to allow for `None`.

Reply:
Done.